packages/adaptive_scaffold/example/lib/main.dart

```diff
@@ -115,6 +115,67 @@ class _MyHomePageState extends State<MyHomePage>
   @override
   Widget build(BuildContext context) {
     const Color iconColor = Color.fromARGB(255, 29, 25, 43);
+    final Widget trailingNavRail = Column(
+      children: <Widget>[
+        const Divider(color: Colors.white, thickness: 1.5),
+        const SizedBox(height: 10),
+        Row(children: <Widget>[
+          const SizedBox(width: 22),
+          Text('Folders',
+              style: TextStyle(fontSize: 13, color: Colors.grey[700]))
+        ]),
+        const SizedBox(height: 22),
+        Row(
+          children: <Widget>[
+            const SizedBox(width: 16),
+            IconButton(
+                onPressed: () {},
+                icon: const Icon(Icons.folder_copy_outlined),
+                iconSize: 21),
+            const SizedBox(width: 21),
+            const Text('Freelance'),
+          ],
+        ),
```

Comment:
You could turn this into a Widget with parameters for text and icon and then replace this and the other instances with that Widget.